apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -240,22 +242,25 @@ public async Task BindNumericEmptyArray()
             new DateTime?[] { });
 
         [SkippableFact]
+        [Trait(Constants.SkipOnEmulator, Constants.Yes)]
         public async Task BindJson()
         {
             Skip.If(_fixture.RunningOnEmulator, "The emulator doesn't yet support the JSON type");
             await TestBindNonNull(SpannerDbType.Json, "{\"key\":\"value\"}", r => r.GetString(0));
         }
 
         [SkippableFact]
+        [Trait(Constants.SkipOnEmulator, Constants.Yes)]
         public async Task BindJsonArray()
         {
             Skip.If(_fixture.RunningOnEmulator, "The emulator doesn't yet support the JSON type");
             await TestBindNonNull(
                 SpannerDbType.ArrayOf(SpannerDbType.Json),
                 new string[] { "{\"key\":\"value\"}", null, "{\"other-key\":\"other-value\"}" });
         }
-
+        
```

Comment:
I will fix the whitespace issues and the commented PgJsonb code as it is not supported yet in the actual PR.